src/modules/emote_menu/components/Tip.jsx

```diff
@@ -40,6 +51,24 @@ function getTipToDisplay() {
     ];
   }
 
+  if (!tips[EmoteMenuTips.EMOTE_MENU_REPLACE_DEFAULT] && emoteMenuValue !== EmoteMenuTypes.ENABLED) {
+    return [
+      EmoteMenuTips.EMOTE_MENU_REPLACE_DEFAULT,
+      formatMessage(
+        {defaultMessage: '<button>Replace</button> the default emote picker button'},
```

Comment:
is this confusing UX? clicking on the link causes a potential destructive behavior. we may want to present the setting to a user at most.